packages/mui-material/scripts/rollup.config.js

```diff
@@ -191,6 +198,25 @@ export default [
       replace({ preventAssignment: true, 'process.env.NODE_ENV': JSON.stringify('development') }),
     ],
   },
+  {
+    input,
+    onwarn,
+    output: {
+      file: 'build/umd/material-ui.emotionless.development.js',
```

Comment:
```suggestion
      file: 'build/umd/material-ui.without-emotion.development.js',
```

`emotionless` sounds strange :) Same for the prod